massa-execution-worker/src/execution.rs

```diff
@@ -196,6 +197,11 @@ impl ExecutionState {
             execution_trail_hash,
         )));
 
+        let cur_execution_version;
```

Comment:
```suggestion
        let cur_execution_version = execution_context.lock().execution_component_version;
```

no ?

Reply:
Indeed, I worried that the lock did not go out of scope and caused a deadlock, but that is unnecessary, thanks!